packages/asc-ui/src/types/Theme.ts

```diff
@@ -103,7 +103,7 @@ export namespace Theme {
       SupportPaletteInterface {}
 
   export type TypographyElementStyle = {
-    color: CSSProp
+    color: CSSProp // This will be deprecated in the future
```

Comment:
Should this not be deprecated now? Best to add an `@deprecated` annotation here so the IDE will pick it up.

Reply:
Will do